fhir-bucket/src/main/resources/stats/postgres/table_sizes.sql

```diff
@@ -0,0 +1,114 @@
+select resource_type || param.nm AS param_table, pg_relation_size('fhirdata.' || resource_type || param.nm)
```

Comment:
Copyright header

Reply:
useful outside of fhir-bucket, and should probably go with performance folder

Reply:
fhir-bucket will be moved to the performance folder anyway. But I agree it would be useful to start building a library of useful SQL for different DB flavors.